### DISCUSSION TITLE
Win 11 - following steps of Script Kit Presentation fails to produce same results

### DISCUSSION THREAD

**Discussion by Ambushfall** [johnlindquist/kit#1058] [Errors] [answered]
Greetings!

I've attempted to follow the tutorial for the project folder listing and opening and was unable to obtain the same results.

I'll post the script below:

```ts
// Preview: docs
// Name: dev-project
import "@johnlindquist/kit";

let projectDir = home('Desktop/projectz/');

let projects = await readdir(projectDir);

let project = await arg('Enter a project name:', projects);

let projectPath = path.resolve(projectDir, project);

await $`code ${projectPath}`;
```

I'll post the result of the actual command running through via image below:

It seems that it has resolved the path name for the CLI in .kenv then launched code with parameters of a filename instead of a directory, it has also included the last backtick in the path.

I understand this is a WIP and would love future versions to support this!

![fileOpenInVsCode](https://user-images.githubusercontent.com/95249049/214652638-9c23396c-3e39-4f4f-96e1-3540444e8a5b.png)

**Comment by Ambushfall** [comment, marked as answer]
In order to have this run on Win it's required to use the following instead

```ts
await edit(projectPath);
```

Located solution by finding the following script and the line at the end.

// Menu: Open Project
// Description: Opens a project in code
// Shortcut: cmd shift .

**Comment by Evyweb** [comment]
Hi guys, as an alternative, you can use "cd(path)" and then "code ." on windows. I use it in my script to open IntelliJ projects.
